BasicHistoryApi/app/src/main/AndroidManifest.xml

```diff
@@ -1,19 +1,19 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!--
-Copyright 2014 Google, Inc.
+  Copyright 2019 Google LLC
```

Comment:
You actually don't need to update the year for existing files (leave them alone generally).